rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -229,7 +229,7 @@ def __iter__(self):
         return self
 
     def __len__(self):
-        return self.iterator_length
+        return self.iterator_length // self.batch_size
```

Comment:
why is this changed?

Reply:
In ToT, The pytorch iterator is returning the number of images for each shard but it should return the number of batches in the loader.